arch/posix/core/posix_core.c

```diff
@@ -42,13 +42,12 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "posix_core.h"
-#include "posix_arch_internal.h"
-#include "posix_soc_if.h"
-#include "kernel_internal.h"
-#include "kernel_structs.h"
-#include "ksched.h"
-#include "kswap.h"
+#include <posix_core.h>
```

Comment:
Nit: (again not going to block the PR because of this one), this "" to <> change would not seem to be related to this commit itself but add quite a bit of noise to it.